releasenotes/notes/40394.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+issue:
+  - 40394
+  - 37622
+  - 35781
+  - 41271
+
+docs:
+  - '[Optimized Design] https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/'
+  - '[Original Design] https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/'
+
+releaseNotes:
+- |
+  **Added** Add feature flags EnableDualStack to support Dual Stack for both istiod and istio-agent 
+  **Added** to support the multiple addresses for single listener by adding additional addresses in LDS configuration
+  **Added** to support the smarter way to pick endpoints based on its IP families by adding extra source addresses in Inbound cluster BindConfig
+  **Enabled** enable the dual stack support in Istio for normal Istio service in service mesh
```

Comment:
I would just make one entry that says "Enabled dual stack support behind feature flag X". The rest is implementation details

Reply:
Sure, I will merge all information into one entry